config/default/core.entity_view_display.node.agency_component.teaser.yml

```diff
@@ -27,13 +27,17 @@ content:
     region: content
 hidden:
   field_agency: true
+  field_agency_comp_abbreviation: true
+  field_agency_comp_telephone: true
   field_agency_component_id: true
   field_email: true
   field_foia_officers: true
+  field_misc: true
   field_portal_submission_format: true
   field_public_liaisons: true
   field_reading_rooms: true
   field_receiver: true
+  field_request_submission_form: true
```

Comment:
@ba66e77 Why are we adding the request form to the teaser view here?

Reply:
yeah, this was inadvertent.  Not sure how it got in there.  Will be fixed in next commit.